### ISSUE TITLE
Complete Unit tests for DatabaseReader

### ISSUE THREAD

**Issue by kermit10000000** [JaroslawWiosna/regionalizer#59]
#### Descrpition

When functions will be splited into smaller ones I will prepare tests scenarios and unit tests for each function. Help wanted not for unit test part but for spliiting functions into smaller ones.

--- 

#### Release affected

Yes. It is part of the 0.3.0 release

---

#### Priority

HIGH - because it must be done before 0.3.0

---

#### Is there possible contributor who could fix the issue

Wojciech Mielczarek

---

**Comment by JaroslawWiosna** [comment]
More about unit test examples:
 - [MatrixLibs](https://github.com/JaroslawWiosna/MatrixLibs)
 - [BigNumber](https://github.com/JaroslawWiosna/BigNumber)

**Comment by kermit10000000** [comment]
some tests like if we have 4 times "|" and not other signs like "*" will not be introduced because they should be tested on the level of database file tests(that currently is on hold because of bug)
